test/bib-marc-test.js

```diff
@@ -17,10 +17,10 @@ describe('Bib Marc Mapping', function () {
 
       var altTitleMapping = mapping.getMapping('Alternative title')
       // right number of alt title mappings:
-      assert.strictEqual(altTitleMapping.paths.length, 5)
+      assert.strictEqual(altTitleMapping.paths.length, 6)
```

Comment:
I assume this needed to be bumped because mappings were added to NYPL-Core#master, which broke this assertion. It's not related to this PR really.